### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by axelssonHakan**

### CODE REVIEW

File: src/components/error-boundary.jsx [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import logo from '../../catwalk.svg';
```
Comment (peol) [inline src/components/error-boundary.jsx]: We should have the logo in the `src/` folder imo, the web app shouldn't pull resources from outside its root folder.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # catwalk
```
Comment (peol) [inline README.md]: You could remove this header too

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
 
+![logo](./catwalk.svg)
```
Comment (peol) [inline README.md]: Instead of 'logo' perhaps 'catwalk' and indexing of the page would still read catwalk

**Comment by hrigner** [comment]
😍

**Pull request merged**